isobus/include/isobus/isobus/nmea2000_fast_packet_protocol.hpp

```diff
@@ -46,16 +46,22 @@ namespace isobus
 		void initialize(CANLibBadge<CANNetworkManager>) override;
 
 		/// @brief Similar to add_parameter_group_number_callback but tells the stack to parse those PGNs as Fast Packet
+		/// @details You may pass a destination `InternalControlFunction`, which will filter for only those messages
+		/// that target this source/destination pair (see https://github.com/ad3154/Isobus-plus-plus/issues/206).
 		/// @param[in] parameterGroupNumber The PGN to parse as fast packet
 		/// @param[in] callback The callback that the stack will call when a matching message is received
 		/// @param[in] parent Generic context variable
-		void register_multipacket_message_callback(std::uint32_t parameterGroupNumber, CANLibCallback callback, void *parent);
+		/// @param[in] destinationFunction An optional internal function destination to filter messages by
+		void register_multipacket_message_callback(std::uint32_t parameterGroupNumber, CANLibCallback callback, void *parent, InternalControlFunction *destinationFunction = nullptr);
```

Comment:
Good plan to add it to fast packet, I probably would have forgotten about that. We probably also need to add some code to its cpp file to handle this new parameter here:
![image](https://user-images.githubusercontent.com/10929341/224343368-7c9814f3-1324-4a5c-bfc2-42771cbd2ce8.png)